### PR TITLE
Add handeye dashboard

### PR DESCRIPTION
**Pull request by RoboticsYY**
This PR intends to add `handeye_dashboard` module, which is the GUI tool for the lightweight handeye calibration. 

Directory structure:
```shell
.
├── config
│   └── Default.perspective
├── data
├── doc
│   └── images
├── images
├── launch
│   └── handeye_dashboard.launch.py
├── package.xml
├── plugin.xml
├── README.md
├── resource
│   └── handeye_dashboard
├── setup.py
└── src
    └── handeye_dashboard
        ├── handeye_calibration.py
        ├── __init__.py
        ├── main.py
        └── plugin.py
```

### CODE REVIEW

File: grasp_utils/handeye_dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  * Install [ROS2 Dashing](https://index.ros.org/doc/ros2/Installation/Dashing/Linux-Install-Debians/)
+* Install [handeye](https://github.com/RoboticsYY/handeye)
+* Install [criutils](https://github.com/RoboticsYY/criutils)
+* Install [baldor](https://github.com/RoboticsYY/baldor)
+* Install [handeye_tf_service](https://github.com/intel/ros2_grasp_library/tree/master/grasp_utils/handeye_tf_service)
+* Install [handeye_target_detection](https://github.com/intel/ros2_grasp_library/tree/master/grasp_utils/handeye_target_detection)
+
+## 2. Build and install
+
+Build with the `ros2_grasp_library` package.
```
Comment (sharronliu) [inline grasp_utils/handeye_dashboard/README.md]: to be specific, just mention "colcon build --symlink-install"

File: grasp_utils/handeye_dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+Please check the result at `~/ros2_ws/install/handeye_dashboard/share/handeye_dashboard/data/camera_robot.txt`:
+
+```yaml
+camera-robot pose:
+  Translation: [-0.032727495589941216, -0.09304065368400717, 0.0003508296697299189]
+  Rotation: in Quaternion [0.9997471812284859, 0.01090594636560865, -0.009141740972837598, -0.0174086912647742]
+
+```
+
```
Comment (sharronliu) [inline grasp_utils/handeye_dashboard/README.md]: To my understanding, here "publish result" suggest "Step 3 use the fourth button to publish..." mentioned above.
Shall we add a way to publish the result without launching the whole GUI? For example, just mentioning
`ros2 run tf2_ros static_transform_publisher <Translation x, y, z> <Rotation x, y, z, w> <from_frame> <to_frame>`

File: grasp_utils/handeye_dashboard/README.md (hunk 1 above)
Comment (RoboticsYY) [reply grasp_utils/handeye_dashboard/README.md]: Done. Added link to the installation instructions of `ros2_grasp_library`.

File: grasp_utils/handeye_dashboard/README.md (hunk 2 above)
Comment (RoboticsYY) [reply grasp_utils/handeye_dashboard/README.md]: Done. Add instructions about how to publish the result in command line.

**Pull request merged**